source/wifi/wifi_hal.c

```diff
@@ -22,6 +22,14 @@
 * Licensed under the BSD-3 license
 */
 
+//Code in rxStatsInfo_callback, txStatsInfo_callback and in chanSurveyInfo_callback originates from:
```

Comment:
Please use the text in BD-1023 on this line:
"Code in rxStatsInfo_callback and other callbacks is credited as follows:"